lux-eye/src/pages/visualizer/Board.tsx

```diff
@@ -108,7 +108,12 @@ function drawTileBackgrounds(ctx: CanvasRenderingContext2D, config: Config, step
     const [canvasX, canvasY] = tileToCanvas(config, { x: board.relicNodes[i][0], y: board.relicNodes[i][1] });
 
     ctx.fillStyle = 'orange';
-    ctx.fillRect(canvasX + config.tileSize / 4, canvasY + config.tileSize / 4, config.tileSize / 2, config.tileSize / 2);
+    ctx.fillRect(
```

Comment:
this is automatically changed due to the `prettier` config in `package.json`

https://github.com/Lux-AI-Challenge/Lux-Design-S3/blob/main/lux-eye/package.json#L154